client/public/index.html

```diff
@@ -45,19 +47,9 @@
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
   <script>
-    document.addEventListener('DOMContentLoaded', function () {
-      var elems = document.querySelectorAll('.sidenav');
-      var instances = M.Sidenav.init(elems, options);
-    });
-
-    // Initialize collapsible (uncomment the lines below if you use the dropdown variation)
-    // var collapsibleElem = document.querySelector('.collapsible');
-    // var collapsibleInstance = M.Collapsible.init(collapsibleElem, options);
-
-    // Or with jQuery
-
     $(document).ready(function () {
       $('.sidenav').sidenav();
+      $('.modal').modal();
```

Comment:
Why do we need jquery? What's the reasoning behind this? you can probably just have all of this in react.

Reply:
Agreed that this may be a valuable removal if you have time to remove now. If you have a bunch and it is tightly coupled you may want to put in ice box. In general, jQuery is not totally dead... But inside frameworks like React is is basically dead:)

Reply:
Removed as well.